draft-ietf-taps-interface.md

```diff
@@ -258,6 +258,169 @@ Errors and other notifications also happen asynchronously on the Connection.
 through Actions and Events in each phase of a Connection, following the phases
 described in {{I-D.ietf-taps-arch}}.
 
+## Usage Examples
+
+The following usage examples illustrate how an application might use a
+Transport Services Interface to
```

Comment:
Editorial nit, feels like this should be structured like:

```
how an application might use a Transport Services Interface to:

- Act as a server, by listening for incoming connections, receiving requests, and sending responses. See {{server-example}}.
```
etc. The main point being we should have a `:` on the dangling sentence, which is filled out by the bullets.

Reply:
Agree, will change.